client/account_number.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/mxenabled/atrium-go/models"
+	"net/http"
+)
+
+func parseAccountNumbersResponse(response *http.Response) ([]*models.AccountNumber, error) {
+	if err := parseResponseErrors(response.StatusCode); err != nil {
+		return nil, err
+	}
+
+	buffer := new(bytes.Buffer)
+	buffer.ReadFrom(response.Body)
+	bufferStr := buffer.String()
+	response.Body.Close()
+
+	if response.StatusCode == 200 {
+		accountNumbersResponse := &models.AccountNumbersResponse{}
+		json.Unmarshal([]byte(bufferStr), accountNumbersResponse)
```

Comment:
`json.Unmarshal` returns an `error`, so you're going to need to check that and return if not `nil`. Same for a few other places in this PR.

Reply:
Otherwise, LGTM

Reply:
Good catch. Error handling has been added.